### PR TITLE
Basic structure for rockets

### PR DESCRIPTION
**Pull request by daudi13**
In this branch I: 
- Created a rocket page
- temporarily disabled the navbar 
- Styled the page using a mobile first approach
- Added items dynamically

_I have commented out the navbar on this branch because it wasn't connected to the routers I'll fix it in the next branch seeing already you have installed routers in your last pull request_

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
-import NavBar from './components/NavBar';
+// import NavBar from './components/NavBar';
```
Comment (Ismailco) [inline src/App.js]:
```suggestion
import NavBar from './components/NavBar';
```
You could leave the changes here, and just add your component "Rocket" to the existing code in the dev branch

File: src/components/Rocket/RocketsCard.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable react/prop-types */
```
Comment (Ismailco) [inline src/components/Rocket/RocketsCard.js:1]: You can import and use Prop-Types instead of ignoring them, they are already installed

**Review by Ismailco — APPROVED** [review]
Hi @daudi13, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
Congratulations! 🎉 

## Highlights

I added some suggestions, you can take them into account

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: src/App.js (hunk 1 above)
Comment (daudi13) [reply src/App.js]: okay I will after I merge thank you.

File: src/components/Rocket/RocketsCard.js (hunk 2 above)
Comment (daudi13) [reply src/components/Rocket/RocketsCard.js:1]: Noted. I just fixed the issue. Thanks man :+1:

**Pull request merged**